NitroxClient/GameLogic/Cyclops.cs

```diff
@@ -1,15 +1,37 @@
-﻿using NitroxClient.Communication.Abstract;
+﻿using System.Collections.Generic;
+using System.Linq;
+using NitroxClient.Communication.Abstract;
+using NitroxClient.GameLogic.Helper;
+using NitroxClient.Unity.Helper;
+using NitroxModel.DataStructures.GameLogic;
+using NitroxModel.DataStructures.Util;
+using NitroxModel.Helper;
+using NitroxModel.Logger;
 using NitroxModel.Packets;
+using UnityEngine;
 
 namespace NitroxClient.GameLogic
 {
     public class Cyclops
     {
         private readonly IPacketSender packetSender;
+        private readonly SimulationOwnership simulationOwnershipManager;
 
-        public Cyclops(IPacketSender packetSender)
+        /// <summary>
+        /// KeyValuePair<Guid, douseAmount>. Used in <see cref="OnFireDoused(Fire, SubRoot, float)"/> to reduce
+        /// the <see cref="CyclopsFireHealthChanged"/> packet spam as fires are being doused. A packet is only sent after
```

Comment:
In a future PR (not this one) we should look into doing the generic rate limiter.  I don't think Marin finished it.